src/signin.js

```diff
@@ -1,91 +1,90 @@
-  if (email === "") {
-    setInputError({ input: emailInput, errorMessage: emailErrorMessage }, "이메일을 입력해주세요.");
-    return;
-  }
-  if (!isEmailValid(email)) {
-    setInputError(
-      { input: emailInput, errorMessage: emailErrorMessage },
-      "올바른 이메일 주소가 아닙니다."
-    );
-    return;
+signinPassword.addEventListener("focusout", function () {
+  if (!signinPassword.value) {
+    // 빈칸일 때
+    errPassword.style.display = "block";
+    errPassword.innerText = "비밀번호를 입력해주세요.";
+    signinPassword.style.borderColor = "var(--red)";
+  } else {
+    errPassword.style.display = "none";
+    signinPassword.style.borderColor = "var(--gray-20)";
   }
-  removeInputError({ input: emailInput, errorMessage: emailErrorMessage });
-}
+});
 
-const passwordInput = document.querySelector("#password");
-const passwordErrorMessage = document.querySelector("#passwordErrorMessage");
-passwordInput.addEventListener("focusout", (event) => validatePasswordInput(event.target.value));
-function validatePasswordInput(password) {
-  if (password === "") {
-    setInputError(
-      { input: passwordInput, errorMessage: passwordErrorMessage },
-      "비밀번호를 입력해주세요."
-    );
-    return;
+function submit() {
+  // e.preventDefault();
+  if (signinEmail.value === TEST_EMAIL && signinPassword.value === TEST_PWD) {
+    alert("로그인 성공");
+    const link = "folder.html";
+    window.location.href = link;
+  } else {
+    if (signinEmail.value !== TEST_EMAIL) {
+      errEmail.style.display = "block";
+      errEmail.innerText = "이메일을 확인해주세요.";
+      signinEmail.style.borderColor = "var(--red)";
+    }
+    if (signinPassword.value !== TEST_PWD) {
+      errPassword.style.display = "block";
+      errPassword.innerText = "비밀번호를 확인해주세요.";
+      signinPassword.style.borderColor = "var(--red)";
+    }
   }
-  removeInputError({ input: passwordInput, errorMessage: passwordErrorMessage });
 }
 
-const passwordToggleButton = document.querySelector("#passwordToggle");
-passwordToggleButton.addEventListener("click", () =>
-  togglePassword(passwordInput, passwordToggleButton)
-);
+// const passwordToggleButton = document.querySelector("#passwordToggle");
+// passwordToggleButton.addEventListener("click", () =>
+//   togglePassword(passwordInput, passwordToggleButton)
+// );
+
+// function togglePassword(input, toggleButton) {
+//   if (input.getAttribute("type") === "password") {
+//     input.setAttribute("type", "text");
+//     toggleButton.getElementsByTagName("img")[0].setAttribute("src", "./images/eye-on.svg");
+//     return;
+//   }
+//   input.setAttribute("type", "password");
+//   toggleButton.getElementsByTagName("img")[0].setAttribute("src", "./images/eye-off.svg");
+// }
+
+const button = document.getElementById("signinBtn");
+button.addEventListener("click", submit);
+
+//키보드 엔터로 제출
+document
+  .getElementById("form")
+  .addEventListener("keydown", function (event) {
+    if (event.key === "Enter") {
+      submit();
+    }
+  });
```

Comment:
form 태그 사용하셨다면 이렇게 사용해보시는 것을 추천드려요.

```suggestion
document
  .getElementById("form")
  .addEventListener("submit", function (event) {
        event.preventDefault();
        submit();
  });
```